src/Sound.js

```diff
@@ -35,13 +35,18 @@ export default function Sound(props) {
   const { isLoggedIn, freeSound } = props;
   const [sound, setSound] = useState({});
   const [packSounds, setPackSounds] = useState([]);
+  const [isPlaying, setIsPlaying] = useState(false);
   const [similarSounds, setSimilarSounds] = useState([]);
   // 0 => loading
   // 1 => loading succeeded
   // 2 => loading failed
   const [loadingState, setLoadingState] = useState(0);
   const { id } = useParams();
 
+  const handlePlayingAndPausing = () => {
+    setIsPlaying(!isPlaying);
```

Comment:
`setIsPlaying` needs to be called when progress has finished. You can hook into this with the `onFinish` prop:

https://github.com/amilajack/react-wavesurfer.js/blob/master/src/index.tsx#L114

Reply:
Working on this right now. Is that to automatically change the button when a sound finishes?

Reply:
Yup, that's correct

Reply:
Sounds good. It appears to work, now. You can see what I think is a fully working commit in [the `add-play-button` branch](https://github.com/amilajack/freesound-player/tree/add-play-button).